src/coreclr/tools/Common/Compiler/DependencyAnalysis/ObjectDataBuilder.cs

```diff
@@ -319,6 +319,9 @@ public void EmitReloc(ISymbolNode symbol, RelocType relocType, int delta = 0)
 
                 case RelocType.IMAGE_REL_BASED_LOONGARCH64_PC:
                 case RelocType.IMAGE_REL_BASED_LOONGARCH64_JIR:
+
+                case RelocType.IMAGE_REL_BASED_RISCV64_PC:
+                case RelocType.IMAGE_REL_BASED_RISCV64_JALR:
```

Comment:
I think `IMAGE_REL_BASED_RISCV64_PC` and `IMAGE_REL_BASED_RISCV64_JALR` has the same behavior in RISCV64. IMO, it is better to merge into one such as  `IMAGE_REL_BASED_RISCV64_AUIPC` You need to update IMAGE_REL_RISCV64_PC and IMAGE_REL_RISCV64_JALR in runtime as well.